### PR TITLE
"Show VLANs" Button in Device

### PR DESCRIPTION
**Pull request by Arkasha99**
### Fixes: <#4126>

according to issue I added "Show VLANs" button to interface table in device viewpage. It works exactly like "Show IPs" button showing VLANs instead. 
The table looks almost the same as VLANs table in interface viewpage. 
The table has been added to interface.html, script is in js/interface_toggles.

### CODE REVIEW

**Review by DanSheps — CHANGES_REQUESTED** [review]
Once you correct these, I will take another look at it and see how it looks on develop.

Keep in mind though, this might significantly increase the query time for interfaces so we need to do thorough testing.

File: netbox/templates/dcim/inc/interface.html [hunk 1]
```diff
@@ -186,10 +186,10 @@
 
 {% with ipaddresses=iface.ip_addresses.all %}
     {% if ipaddresses %}
-        <tr class="ipaddresses">
-            {# Placeholder #}
```
Comment (Shm013) [inline netbox/templates/dcim/inc/interface.html]: extraspaces

File: netbox/project-static/js/interface_toggles.js [hunk 2]
```diff
@@ -2,9 +2,23 @@
 $('button.toggle-ips').click(function() {
     var selected = $(this).attr('selected');
     if (selected) {
-        $('#interfaces_table tr.interface:visible + tr.ipaddresses').hide();
+        $('.ipaddresses').hide();
```
[the remark this answers is not in the document]
Comment (Arkasha99) [reply netbox/project-static/js/interface_toggles.js]: At first i was trying to make the VLANs table working like IPs table (using the tr and its class to hide rows) but it didn't work properly. The script for hiding VLANs table didn't work at all. When I was clicking "show VLANs" Ihad no response (when I had IP and VLAN table at the same time only IP row was hiding). So the reason of changing the script and td class was the only way (as for me) to make it work, and it worked great.

**Comment by jeremystretch** [comment]
I have marked #4126 as blocked by #4193. Per my comment [here](https://github.com/netbox-community/netbox/issues/4126#issuecomment-587501933) cannot accept any further modifications to the device interfaces list template until it has been refactored substantially.

FYI this is why the pull request template says to wait until an issue has been accepted before submitting any work. Thank you for your contribution, but unfortunately we cannot accept it at this time.

**Pull request closed without merging**